app/assets/javascripts/admin/addon/templates/components/site-settings/upload.hbs

```diff
@@ -1,2 +1,8 @@
-{{site-settings-image-uploader imageUrl=value placeholderUrl=setting.placeholder type="site_setting"}}
```

Comment:
The site-settings-image-uploader solely existed to pass `for_site_setting=true` to the queryString for image uploads for site settings. This is replaced by passing the `queryParams` object into `uppy-image-uploader`